src/openvic-simulation/misc/Modifier.cpp

```diff
@@ -367,20 +361,32 @@ bool ModifierManager::parse_scripts(GameManager const& game_manager) {
 	return ret;
 }
 
+static const case_insensitive_string_set_t no_effect_modifiers {
+	"boost_strongest_party",	"poor_savings_modifier", "poor_life_needs",
+	"poor_everyday_needs",		"poor_luxury_needs",	 "middle_life_needs",
+	"middle_everyday_needs",	"middle_luxury_needs",	 "rich_life_needs",
+	"rich_everyday_needs",		"rich_luxury_needs",	 "local_artisan_input",
+	"local_artisan_throughput", "local_artisan_output",	 "artisan_input",
+	"artisan_throughput",		"artisan_output",		 "import_cost"
+};
```

Comment:
This can go inside `_modifier_effect_callback` since it's not needed anywhere else in the code